### PR TITLE
Tesha/fqdn filtering

### PR DESCRIPTION
**Pull request by tejasshah7**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [x] I have reviewed the [documentation](https://github.com/Azure/azure-rest-api-specs#basics) for the workflow.
- [x] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [x] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [x] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [ ] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

**Review by anton-evseev — CHANGES_REQUESTED** [review]
Just to be sure: are those two new properties settable by the customer? Or are they read-only?

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json [hunk 1]
```diff
@@ -715,6 +715,10 @@
           "type": "string",
           "description": "The translated address for this NAT rule."
         },
+        "translatedFqdn": {
```
[the remark this answers is not in the document]
Comment (tejasshah7) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json]: Sure moved both fields.

File: specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json [hunk 2]
```diff
@@ -827,6 +831,13 @@
             "type": "string"
           }
         },
+        "destinationFqdns": {
```
[the remark this answers is not in the document]
Comment (tejasshah7) [reply specification/network/resource-manager/Microsoft.Network/stable/2019-09-01/azureFirewall.json]: Done.

**Comment by tejasshah7** [comment]
> Just to be sure: are those two new properties settable by the customer? Or are they read-only?

These can be set by customers.

**Review by anton-evseev — CHANGES_REQUESTED** [review]
You've updated order in examples, I was talking about spec file

**Comment by tejasshah7** [comment]
> You've updated order in examples, I was talking about spec file

Sorry my bad. Updated the spec file now.

**Review by anton-evseev — APPROVED** [review]

**Review by KrisBash — APPROVED** [review]
LGTM

**Review by sergey-shandar — APPROVED** [review]

**Pull request merged**